service/lib/agama/dbus/clients/storage.rb

```diff
@@ -67,7 +67,8 @@ def finish
         #
         # @return [Hash]
         def config
-          serialized_config = dbus_object.GetConfig
+          # Use storage iface to avoid collision with bootloader iface
```

Comment:
Interesting. I wonder if it worth to always use the corresponding iface to avoid more errors like the one this commit fixes and get rid of specific comment. 

Something to be discussed, not a blocker for this PR.

Reply:
yeah, it is zbus way to always use combination of service, interface and object. On other hand ruby-dbus use different way and allow access throught interface, but for simplicity if method in unique, it is added directly to object. My original fix change dbus_object to always return interface, but then mixins start failing as it uses some methods that is not in interface. (during test e.g. this mixins failing https://github.com/agama-project/agama/blob/master/service/lib/agama/dbus/clients/with_service_status.rb#L43 ).
Maybe it will be better to have beside dbus_object also dbus_interface method on which we should call all needed storage methods ( that is idea that I get after thinking a bit more about it after I have working solution and not enough time to test different one ).